swag/client/yfu.py

```diff
@@ -0,0 +1,54 @@
+from disnake.ext import commands
+import disnake
+from swag.id import UserId
+from swag.blocks.yfu_blocks import YfuGenerationBlock
+from utils import GUILD_ID
+
+from .ui.IHS_toolkit import sort_yfus_id
+from .ui.yfu_view import YfuEmbed, YfuNavigation, YfuExchange
+
+
+class YfuCommand(commands.Cog):
```

Comment:
Do we really need a Cog, or is it a choice? What is the motivation? What is a Cog? Where am I? Who am I? Where is Bryan?

Reply:
Yes, we must use Cog.

Cog is the only way to create `slash_command` in a different python file where the Bot is instanciated.

If we don't use Cog, we are forced to declare ALL off our command in `main.py`, and you don't want that c:

Idfk what the word Cog means (I searched, I promise) but its mainly role is what I described just above.